homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -304,7 +308,21 @@ async def async_create_miio_device_and_coordinator(
         device = AirPurifier(host, token)
     elif model.startswith("zhimi.airfresh."):
         device = AirFresh(host, token)
-    elif model in MODELS_VACUUM:
+    elif (
+        model in MODELS_VACUUM
+        or model.startswith(ROBOROCK_GENERIC)
+        or model.startswith(ROCKROBO_GENERIC)
+    ):
+        if model not in MODELS_VACUUM and (
+            model.startswith(ROBOROCK_GENERIC) or model.startswith(ROCKROBO_GENERIC)
+        ):
+            _LOGGER.warning(
+                "Vacuum model %s is not officially supported, please create an issue at "
+                "https://github.com/rytilahti/python-miio/issues and provide the model name if it works as "
+                "expected",
```

Comment:
I'd rather set this as `debug` for now. I'm all for gathering a (model, feature) mapping to the backend lib where it belongs, but in the current state it makes no sense to obtain raw data like "model X is supported" without any auxiliary information about what features, sensors etc. are supported.

Reply:
With the way the sensors are setup now, only supported ones are created, the rest are dropped due to the None check. So would providing "which sensors have been created" not enough info? 🤔

Reply:
Hmmm. I think this warning can be removed altogether. The backend lib has already a warning (https://github.com/rytilahti/python-miio/blob/master/miio/device.py#L152) which will inform the users when the python-miio gets its next release.

The users who are wondering about missing sensors (and are capable of enabling the debug logger) will likely let us know when the whitelisting approach (i.e., adding extra sensors only on known models) is not working.

Or what do you think?

Reply:
Has this warning been there forever? And folks are not reporting models? Are we sure that this warning is actually being seen? 

But agree, if that warning actually works then there is no need to add it another time in HA. Will alter code tomo morning.

Reply:
No, that warning is not yet in any released python-miio releases iirc, so that's why there have been no reports. But as long as the homeassistant integration is only supporting roborock vacuums, it's not a big deal to fallback to basic support and not logging a warning as that's how it's been for a long time already :-)

Reply:
Alright, removed the warning log.